vision.md

```diff
@@ -0,0 +1,115 @@
+- Supporting IoT and other Northbound connectivity 
+	- Vendor agnostic connectivtiy of the data plane and control plane, analytics and service/app orchestration from various IoT platforms is required for all future IoT use-cases, as hyperscaler platforms and end-customer vendor preferences might vary. 
+-   Edge devices as deployment option for IoT services and applications 
+	- There is increasing need for device specifc applications close to the device e.g. device configuration, control logic, local monitoring, here a flexible software management framework is needed which is independent from the preferred sw artefact type due to different hardware, OS variants and package managers. 
+-   There is an emerging needs for Edge analytics such as data filtering, pre-aggregation, ML model execution. 
+-   Edge devices as configuration/management interface for asset - local/remote UI for asset configuration/management including software and firmware management of underlying systems to keep them secure and up-to date
+
+## What is it and why thin-edge.io is a game changer?
+
+To address the problems above, we created thin-edge.io as a framework for lightweight IoT gateways, PLC’s, routers and other embedded devices which require integration and interoperability with IoT platforms.
+
+The framework includes modules for cloud connectivity, data mapping, device management, intra-edge communication, and certificate management, all aspects and challenges our target personas need to address.
+
+In combination with IoT platforms, thin-edge.io is a foundation for enabling the devices with the following capabilities:
+
+- support for effortless and secure edge device lifecycle management for single and device fleets
+- support for low-touch provisioning of thin edge devices
+- support for local and remote configuration
+- support for local and remote maintenance including remote access (monitoring/troubleshooting),
+- decommissioning of thin devices (e.g. for security compromosed or end of life devices)
+
+Based on the challenge to capture both, the OT and IT persona needs, thin-edge.io is focused on following design principles:
+
+- providing ready-to-use components available on wide-range of hardware, embedded linux varaints (thin edge layer)
+- allowing control and orchestration from IoT (device management)  platforms
+- effortless and secure edge software management for different software artefact types
+- support effortless and secure edge analytics execution/ management for different analytics artefact types and runtimes
+
+**Why are we uniquely positioned to “win” this?**
+
+thin-edge.io offers a unique approach to unify the needs of both the IT and OT world by offering a platform design focused on efficiency robustness and security while offering highest level of extensibility and wider range of hardware.
```

Comment:
```suggestion
thin-edge.io offers a unique approach to unify the needs of both the IT and OT world by offering a platform design focused on efficiency robustness and security while offering the highest level of extensibility and wide range of hardware.
```

Reply:
Missing comma between "efficiency" and "robustness"!

Also: We're offering hardware? I guess something else was meant here, wasn't it? :laughing: